blueprints/quarkus/001_installing_jhipster_quarkus.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Installing JHipster Quarkus
+sitemap:
+priority: 0.5
+lastmod: 2021-01-07T08:40:00-00:00
+---
+
+# Installing JHipster Quarkus
+
+Before using JHipster Quarkus, you have to prepare your environment (Java, Node) and install JHipster.
+
+We recommend to use the JHipster version defined in the JHipster Quarkus blueprint.
+
+| JHipster Quarkus | JHipster |
+|------------------|----------|
+| v0.2.1           | v6.10.5  |
```

Comment:
Note that if we use the jhipster-quarkus command we don't need to install JHipster a part. The blueprint already contains the right JHipster version.

Reply:
Should I remove this table?

Reply:
It's because in the doc, these is `jhipster --blueprints quarkus`

Reply:
I don't think so because it's to have the matrix of compatibility. Maybe we just should dive into details and explain you can use the command to avoid installing JHipster and use the given one.